### PR TITLE
bug fixed : Changed the callout type from tip to warning

### PR DESCRIPTION
**Pull request by gantanikhilraj**
closes issue #6604

### CODE REVIEW

**Review by withshubh — CHANGES_REQUESTED** [review]
- Please keep the `warning` text lowercase
- [All the versions](https://github.com/ToolJet/ToolJet/tree/develop/docs/versioned_docs?rgh-link-date=2023-05-30T10%3A43%3A10Z) from 1.x.x to 2.6.0 and the base [docs/docs](https://github.com/ToolJet/ToolJet/tree/develop/docs/docs?rgh-link-date=2023-05-30T10%3A43%3A10Z) needs this change, you have only updated the `1.x.x`

**Comment by gantanikhilraj** [comment]
@withshubh , Updated, please let me know if there are any changes to make ?

File: docs/versioned_docs/version-1.x.x/contributing-guide/setup/docker.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ docker-compose version 1.26.2, build eefe0d31
 
 ## Setting up
 
-:::tip
+:::WARNING
```
Comment (withshubh) [inline docs/versioned_docs/version-1.x.x/contributing-guide/setup/docker.md:34]: I think you missed this, please make lowercase

**Review by withshubh — CHANGES_REQUESTED** [review]
Please make the change suggested!

**Review by withshubh — APPROVED** [review]
Thanks for contributing to ToolJet! 🚀

**Pull request merged**